### PR TITLE
Disable the bidirection edges.

### LINKED ISSUE #3505: match get vertices error
**Please check the FAQ documentation before raising an issue**

**Describe the bug (__required__)**

```
(root@nebula) [basketballplayer]> MATCH ()-[e:follow]-() RETURN e LIMIT 10
+----------------------------------------------------+
| e                                                  |
+----------------------------------------------------+
| [:follow "player101"->"player102" @0 {degree: 90}] |
| [:follow "player103"->"player102" @0 {degree: 70}] |
| [:follow "player135"->"player102" @0 {degree: 80}] |
| [:follow "player113"->"player106" @0 {degree: 99}] |
| [:follow "player114"->"player115" @0 {degree: 90}] |
| [:follow "player138"->"player115" @0 {degree: 90}] |
+----------------------------------------------------+
Got 6 rows (time spent 2789/3468 us)
```

There are more than 10 edges, but don't return so many rows.

**Your Environments (__required__)**

* OS: `uname -a`
* Compiler: `g++ --version` or `clang++ --version`
* CPU: `lscpu`
* Commit id (e.g. e6ddeb293269d6fb9621358039eb8c554b34bf43)

**How To Reproduce(__required__)**

1. Step 1
2. Step 2
3. Step 3

**Expected behavior**

**Additional context**

### PR DESCRIPTION
**Pull request by Shylock-Hg**
#### What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Which issue(s)/PR(s) this PR relates to?
Close #3505 

  
#### Checklist:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Review by CPWstatic — APPROVED** [review]

File: tests/tck/features/match/Scan.feature [hunk 1]
```diff
@@ -155,3 +155,17 @@ Feature: Match seek by scan
...
+      LIMIT 3
+      """
+    Then a ExecutionError should be raised at runtime: Scan vertices must specify limit number.
+    When executing query:
+      """
+      MATCH ()-[e]-()
+      RETURN type(e) AS Type, e.start_year AS StartYear, e.end_year AS EndYear
+      LIMIT 3
+      """
+    Then a ExecutionError should be raised at runtime: Scan vertices must specify limit number.
```
Comment (yixinglu) [inline tests/tck/features/match/Scan.feature:171]: It's confused to throw the `scan vertices` error, It's obviously `scan edge` in the query.

Comment (Shylock-Hg) [reply tests/tck/features/match/Scan.feature:171]: This plan start from ScanVertices, and transformed to ScanEdges in optimizer, but in this case transform failed, so result in ScanVertices failed.

File: tests/tck/features/match/Scan.feature [hunk 2]
```diff
@@ -155,3 +155,17 @@ Feature: Match seek by scan
       LIMIT 3
       """
     Then a ExecutionError should be raised at runtime: Scan vertices must specify limit number.
+    When executing query:
+      """
+      MATCH ()-[e:is_teacher]-()
+      RETURN type(e) AS Type, e.start_year AS StartYear, e.end_year AS EndYear
+      LIMIT 3
+      """
+    Then a ExecutionError should be raised at runtime: Scan vertices must specify limit number.
```
Comment (jievince) [inline tests/tck/features/match/Scan.feature:164]: The error msg really confuses me. What's the correct way to write this sentence?

Comment (Shylock-Hg) [reply tests/tck/features/match/Scan.feature:164]:
```
MATCH ()-[e:is_teacher]->()
      RETURN type(e) AS Type, e.start_year AS StartYear, e.end_year AS EndYear
      LIMIT 3
```

Comment (Shylock-Hg) [reply tests/tck/features/match/Scan.feature:164]: Don't allow the bidirection.

**Review by yixinglu — APPROVED** [review]

**Pull request merged**